ha/core/system_health/system_health_manager.py

```diff
@@ -57,3 +60,21 @@ def key_exists(self, key: str) -> bool:
         if self._store.key_exists(key):
             return True
         return False
+
+    def parse_key(self, lookup, match_with, prefix):
+        """
+        lookup: cvg
+        match_with: {"node": "1", "device": "/dev/sdd"}
+        prefix: cluster_key
+        """
+        match_str = [ _DELIM.join(i) for i in match_with.items() ]
+        values = []
+        keys = self._store.get_keys(prefix)
+        for key in keys:
+            if all(x in key for x in match_str):
+                pattern = f".*{lookup}{_DELIM}(.*){_DELIM}.*"
+                matched_key = re.search(r"%s" % pattern, key)
+                if matched_key:
```

Comment:
Can you please add example here for matched key.

Reply:
Done.

Reply:
Thanks!